xchange-btcmarkets/src/main/java/org/knowm/xchange/btcmarkets/dto/BTCMarketsException.java

```diff
@@ -6,22 +6,31 @@
 
 public class BTCMarketsException extends HttpStatusExceptionSupport {
 
-  private final Boolean success;
-  private final Integer errorCode;
-  private final String clientRequestId;
-  private final Long id;
-  private final List<BTCMarketsException> responses;
+  public final Boolean success;
+  public final Integer errorCode;
+  public final String errorMessage;
+  public final String clientRequestId;
+  public final Long id;
+  public final List<BTCMarketsException> responses;
 
   public BTCMarketsException(
       @JsonProperty("success") Boolean success,
       @JsonProperty("errorMessage") String errorMessage,
       @JsonProperty("errorCode") Integer errorCode,
       @JsonProperty("clientRequestId") String clientRequestId,
       @JsonProperty("id") Long id,
-      @JsonProperty("responses") List<BTCMarketsException> responses) {
+      @JsonProperty("responses") List<BTCMarketsException> responses,
+      // V3
```

Comment:
use different exception class for that pls, rename this one to BTCMarketsLegacyException or add BTCMarketsExceptionV3